Emby.Server.Implementations/Services/ServicePath.cs

```diff
@@ -190,21 +198,12 @@ public RestPath(Func<Type, object> createInstanceFn, Func<Type, Func<string, obj
                     StringComparer.OrdinalIgnoreCase);
         }
 
-        internal static string[] IgnoreAttributesNamed = new[]
-        {
-            "IgnoreDataMemberAttribute",
```

Comment:
why is this no longer needed? I see you transferred `JsonIgnoreAttribute` to a new position

Reply:
The IgnoreDataMemberAttribute got removed, and I moved the field up above the ctor.